alf/utils/conditional_ops.py

```diff
@@ -62,58 +62,26 @@ def conditional_update(target, cond, func, *args, **kwargs):
     Returns:
         nest with the same structure and shape as target.
     """
-    # shape of indices from where() is [batch_size,1], which is what scatter_nd
-    # needs
-    scatter_indices = tf.where(cond)
-    scope = get_current_scope()
+    # shape of torch.where() is a tuple (indices, )
```

Comment:
shape is not a tuple.
The return value of torch.where() is a tuple (indices, )

Reply:
I guess here 'shape' means in the general sense (what it looks like). I can update it.